utilities/pipelines/publish/helper/Get-ModuleTargetVersion.ps1

```diff
@@ -11,6 +11,9 @@ Builds target version as major.minor.patch
 .PARAMETER ModuleFolderPath
 Mandatory. Path to the main/parent module folder.
 
+.PARAMETER CompareJsonVersion
```

Comment:
Same comment, no change to the publishing logic is required

Reply:
Agreed. This only seems to be relevant for the static validation - hence it should also only live in a function used by it.

Reply:
Resolving as outdated